guidelines/index.html

```diff
@@ -1433,6 +1441,10 @@
 					<p>translation of one language, generally a spoken language, into a <a>sign language</a></p>
 					<p>True sign languages are independent languages that are unrelated to the spoken language(s) of the same country or region.</p>
 				</dd>
+        <dt><dfn>Significant animation</dfn></dt>
+				<dd>
+					<p>animation which continues for more than 3 seconds, or is synchronized with a user action, and affects more than 1/3 of the viewport.</p>
```

Comment:
@patrickhlauke  I updated the text at [the top of this page](https://github.com/w3c/wcag21/pull/96#issue-204047227) with (e.g. parallax scrolling effects).  I will update the pull request in GitHub and after I learn out how :-) Thank you!